{
  throw new RuntimeException("FAILED ASSERTION");
}
